### DISCUSSION TITLE
AdMob Documentation not clear on unitId

### DISCUSSION THREAD

**Discussion by xplodeart** [invertase/react-native-firebase#4212] [Q&A]
Hello,

As we follow the documentation for AdMob usage, it says to create a firebase.json file in the root of the project and add the android and ios App IDs in that file, but when actually creating a BannerAd, the unitId is initialized from string.

My question is do we really need that file, or can we simply pass the unitId as a prop without even adding the firebase.json file, for example fetching this value from remote-config and passing it as a prop to the BannerAd component?

**Comment by mikehardy** [comment]
Have you tried? In my experience the app crashes on startup if you don't put the admob id in your plist and AndroidManifest.xml, which this module will helpfully do for you if you use the firebase.json

But feel free to try yourself!

I've only had success using firebase.json, like so:

https://github.com/mikehardy/rnfbdemo/blob/9e80bacc9b62961f89246eea82dd956f4f8660f8/make-demo.sh#L139-L144

**Comment by xplodeart** [comment]
Thanks, will give it a shot, just wanted to know which one takes effect, but seems like this is only for the native part of the configuration and later can be overwritten.
